jwplayer-appletv-web-app/js/ConfigLoader.js

```diff
@@ -59,6 +59,13 @@ function ConfigLoader() {
   }
 
   function _configLoaded(loadedConfig) {
+    if (loadedConfig.modules) {
```

Comment:
Might want to do some type checking here.  `loadedConfig.modules` must be an array of strings.